### ISSUE TITLE
Suggestion: Can you add inverted shoulder offset adjustments for zoom levels?

### ISSUE THREAD

**Issue by TheArchitect01** [mpstark/DynamicCam#76]
I am trying to re-create a God of War-like camera view while below a certain zoom level and have the default centered view while above a certain zoom level in WoW. Sadly the addon doesn't support that yet. Can you perhaps add it when you have time?

It would be basically just the existing zoom level adjustments but inverted :)

**Comment by LudiusMaximus** [comment]
Absolutely, this is definitely something I would want as well. My ideal would be a custom shoulder offest for arbitrarily many zoom levels. As soon as I find time, I will implement this.

**Comment by TheArchitect01** [comment]
That's nice to hear. Take your time.

I am loving the addon so far. Gives WoW really a different feeling.

**Comment by gizzmo** [comment]
I too would like this option. When you're zoomed in close, the shoulder offset should be max, but as you zoom our, decrease value so the offset is 0 at a set zoom.

**Comment by LudiusMaximus** [comment]
Now, it is possible to setup every cvar (including shoulder offset) based on current zoom level!

**Comment by TheArchitect01** [comment]
I just tested it. Awesome! Thanks for adding :D
    Am Donnerstag, 5. März 2026 um 00:52:00 MEZ hat LudiusMaximus ***@***.***> Folgendes geschrieben:  
 
 LudiusMaximus left a comment (mpstark/DynamicCam#76)
Now, it is possible to setup every cvar (including shoulder offset) based on current zoom level!
